rakelib/release.rb

```diff
@@ -3,6 +3,11 @@ def git_directory
   File.directory?(git_dir) && git_dir
 end
 
+def in_git?
+  git_dir = File.expand_path "../../.git", __FILE__
+  File.exist?(git_dir) && git_dir
```

Comment:
@YorickPeterse I'm assuming it's just copied from above. The code in `git_directory` does that to pass the path back. A multi-clause conditional would have made that more clear. Since `#in_git?` looks like it's only intended to be a predicate, passing the path back doesn't appear needed, but it doesn't hurt either. Since this code will all be rewritten in bash soon-ish, it doesn't matter.

I'll merge and if you want to clean it up, feel free.